client/src/game/star.js

```diff
@@ -411,40 +420,42 @@ class Star extends EventEmitter {
 
       for (let i = 0; i < planetCount; i++) {
         let planetContainer = new PIXI.Container();
-
+    
         let distanceToStar = 15 + 5 * i;
         let planetSize =
-          (Math.floor(Math.abs(this.data.location.y) + distanceToStar) % 1.5) +
-          0.5;
-
+            (Math.floor(Math.abs(this.data.location.y) + distanceToStar) % 1.5) +
```

Comment:
Since I'm here, it could be fun randomly generate the size of the planet as well.

The original code could be biased since our home solar system has small rocky planets closer to the sun. However this is _not_ the norm on a cosmological scale, and rather a rare proof of a finely tuned solar system to support life on earth.

Reply:
That's a great idea. I'll add another feature in the issue section to push this pull request to the main one once we figure out the planet names and colors.